tests/test_product.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from buy_together_app import models
+# from buy_together_app import views
+
+QR = "Q5o76MbdiNbXprNEnHfpcGWFp1CMF8XY"
+NAME = "Banana"
+DESCRIPTION = "Good!"
+
+
+@pytest.fixture
+def product0():
+    return models.Product(qr_code=QR, product_name=NAME, description=DESCRIPTION)
+
+
+class TestProductModel:
+    def test_new_product(self, product0):
+        assert product0.qr_code == QR
+        assert product0.product_name == NAME
+        assert product0.description == DESCRIPTION
+
+    @pytest.mark.django_db()
+    def test_save_product(self, product0):
+        product0.save()
+        # print(models.Product.objects.all()[0].product_name)
+        # assert False
+        assert product0 in models.Product.objects.all()
```

Comment:
I think you should also assert that the product is not there before you save.

Reply:
Somehow make sure that adding the product actually added it and it did not exist beforehand (you can add it after deleting if you prefer).

Reply:
done